apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx

```diff
@@ -57,7 +64,116 @@ function PinButton({ unpin, onClick }: { unpin?: boolean; onClick: () => void })
 	);
 }
 
-function MyTokens({
+function TokenRowSuiToUSDC({ suiBalance }: { suiBalance: BigInt }) {
+	const { rawValue } = useSuiBalanceInUSDC(
+		new BigNumber(suiBalance.toString()).shiftedBy(-1 * SUI_DECIMALS),
+	);
+
+	return (
+		<Text variant="subtitle" color="steel-dark" weight="medium">
+			{getUSDCurrency(rawValue)}
+		</Text>
+	);
+}
+
+function TokenRowButton({
+	coinBalance,
+	page,
+	copy,
+}: {
+	coinBalance: CoinBalanceType;
+	page: 'send' | 'swap';
+	copy: string;
+}) {
+	return (
+		<ButtonOrLink
+			to={getSendOrSwapUrl(page, coinBalance.coinType)}
+			onClick={
+				page === 'send'
+					? () =>
+							ampli.selectedCoin({
+								coinType: coinBalance.coinType,
+								totalBalance: Number(BigInt(coinBalance.totalBalance) / MIST_PER_SUI),
+							})
+					: undefined
+			}
+			key={coinBalance.coinType}
+			className="no-underline text-subtitle font-medium text-steel hover:font-semibold hover:text-hero"
+		>
+			{copy}
+		</ButtonOrLink>
+	);
+}
+
+export function TokenRow({
+	coinBalance,
+	renderActions,
+	onClick,
+	as: Tag = 'div',
+	borderBottom,
+}: {
+	coinBalance: CoinBalanceType;
+	renderActions?: boolean;
+	as?: 'div' | 'button';
+	onClick?: () => void;
+	borderBottom?: boolean;
+}) {
+	const coinType = coinBalance.coinType;
+	const balance = BigInt(coinBalance.totalBalance);
+	const [formatted, symbol] = useFormatCoin(balance, coinType);
+	const isButton = Tag === 'button';
+	const isSui = coinType === SUI_TYPE_ARG;
+
+	return (
+		<Tag
+			className={clsx(
+				'flex py-3 pl-1.5 pr-2 rounded hover:bg-sui/10 items-center border-solid border-t-transparent border-x-transparent',
+				isButton && 'bg-transparent hover:cursor-pointer',
```

Comment:
Why not just always apply these styles?

Reply:
`bg-transparent` is in base class, I am keeping cursor-pointer for button only